packages/grid/src/Grid.tsx

```diff
@@ -99,7 +102,7 @@ export interface GridProps<T = any> {
   /**
    * Should return unique string for a given row data object.
    * If rowData is sparse then this function should work with undefined row data
-   * objects and create keys based on row index.
+   * objects and create keys based on row index. `(row: T, index: number) => string`
```

Comment:
As this is a defined type, does it need to be copied here. Worth checking whether the generated documentation will include the actual `RowKeyGetter<T> definition.

Reply:
@steveukx Grid props table in Grid docs is generated using Storybook <ArgsTable /> API. The way we use <ArgsTable />, Storybook infers data automatically from component and in our Grid component, we have our props `rowKeyGetter` typed as `RowKeyGetter<T>`; To make our props table generate the actual definition as I understand, will require us to update its type to `(row: T, index: number) => string` in Grid component. That's not as ideal hence why we've added it in the description instead 

ps. please let me know if anyone knows of a workaround